aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -62,29 +144,19 @@ module AptosFramework::Token {
     /// Set of data sent to the event stream during a withdrawal
     struct WithdrawEvent has drop, store {
         id: TokenId,
-        amount: u64,
-    }
-
-    /// create collection event with creator address and collection name
-    struct CreateCollectionEvent has drop, store {
-        creator: address,
-        collection_name: ASCII::String,
-        uri: ASCII::String,
-        description: ASCII::String,
-        maximum: Option<u64>,
+        amount: u64
     }
 
     /// token creation event id of token created
-    struct CreateTokenEvent has drop, store {
+    struct CreateTokenEvent has copy, store, drop {
         id: TokenId,
-        token_data: TokenData,
-        initial_balance: u64,
+        initial_balance: u64
     }
 
     /// mint token event. This event triggered when creator adds more supply to existing token
     struct MintTokenEvent has drop, store {
         id: TokenId,
-        amount: u64,
+        amount: u64
```

Comment:
nit ,

Reply:
I update this based on the comment. But not sure why having , at the end